### ISSUE TITLE
TypeError: workbox.strategies[options.offlineStrategy] is not a constructor

### ISSUE THREAD

**Issue by harish0507** [nuxt-community/pwa-module#365]
offlineStrategy options is not available in generated `sw.js` file

<img width="1280" alt="Screenshot 2020-10-07 at 1 56 45 PM" src="https://user-images.githubusercontent.com/14799952/95307623-82facd00-08a6-11eb-9dad-f4e7301f793a.png">

Nuxt version: 2.14.6
Nuxt PWA version: 3.1.0

```
// nuxt.config.js
pwa: {
	workbox: {
		dev: true,
		config: process.env.NODE_ENV !== 'development' ? false : {
			debug: process.env.NODE_ENV === 'development'
		},
		cleanupOutdatedCaches: true,
		offline: false,
		runtimeCaching: [
			{
				urlPattern: 'https://(s3-|s3.)?(.*).amazonaws.com/.*',
				handler: 'cacheFirst',
				strategyOptions: {
					cacheName: 's3',
					cacheExpiration: {
						maxEntries: 100,
						maxAgeSeconds: 3600
					}
				}
			},
			{
				urlPattern: '/images/.*',
				handler: 'cacheFirst',
				strategyOptions: {
					cacheName: 'images',
					cacheExpiration: {
						maxEntries: 20,
						maxAgeSeconds: 3600
					}
				}
			},
			{
				urlPattern: '/api/.*',
				handler: 'networkOnly'
			}
		]
	}
}
```

### Proposed Solution

Seems like offlineStrategy options is missing in `@nuxtjs/pwa/lib/workbox/module.js`

```
// Add sw.js
if (options.swTemplate) {
    await this.addTemplate({
        src: options.swTemplate,
        fileName: options.swDest,
        options: {
            swOptions: pick(options, [
                'workboxURL',
                'importScripts',
                'config',
                'cacheNames',
                'clientsClaim',
                'skipWaiting',
                'cleanupOutdatedCaches',
                'offlineAnalytics',
                'preCaching',
                'runtimeCaching',
                'offlinePage',
                'offlineStrategy' // Added missing option
            ]),
            routingExtensions: await readJSFiles.call(this, options.routingExtensions),
            cachingExtensions: await readJSFiles.call(this, options.cachingExtensions),
            workboxExtensions: await readJSFiles.call(this, options.workboxExtensions)
        }
    })
}
```

**Comment by simllll** [comment]
The code block looked like this before:
 // Add sw.js
 ```
   if (options.swTemplate) {
      await this.addTemplate({
        src: options.swTemplate,
        fileName: options.swDest,
        options: {
          ...options,
          routingExtensions: await readJSFiles.call(this, options.routingExtensions),
          cachingExtensions: await readJSFiles.call(this, options.cachingExtensions),
          workboxExtensions: await readJSFiles.call(this, options.workboxExtensions)
        }
      })
    }
```

therefore it seems legit that it worked before, but doesn't now when pick is used.

Breaks in 3.1.0

**Comment by victorgarciaesgi** [comment]
I have this error too!

**Comment by pi0** [comment]
Should be fixed by [v3.1.1](https://github.com/nuxt-community/pwa-module/releases/tag/v3.1.1)
